static/src/stylesheets/module/commercial/paidfor/_paidfor-popup.scss

```diff
@@ -1,12 +1,15 @@
-.paidfor-popup {
+.popup--paidfor {
     background: guss-colour(neutral-1, $pasteup-palette);
     box-shadow: none;
-    color: #ffffff;
-    transform: translate(-50%, 0);
     left: 50%;
     top: 100%;
-    width: 240px;
     padding: 16px;
+}
+
+.paidfor-popup {
+    color: #ffffff;
+    transform: translate(-50%, 0);
+    width: 240px;
```

Comment:
they could, I didn't know how far to go, especially as there are paidfor-popup__<etc> classes as well and I wasn't sure if I should change them and what I should change to if so